### PR TITLE
Added rehydrate option to stop propagation listener

### PR DESCRIPTION
**Pull request by wachterjohannes**
**tasks:**
- [x] test coverage
- [ ] gather feedback for my changes

**informations:**

| q | a |
| --- | --- |
| Fixed tickets | none |
| BC breaks | none |
| Documentation PR | none |

### CODE REVIEW

File: tests/Unit/Subscriber/Core/RegistratorSubscriberTest.php [hunk 1]
```diff
@@ -126,6 +129,26 @@ public function testStopPropagation()
     }
 
     /**
+     * It should notstop propagation if the document is loaded with rehydrate option.
```
Comment (dantleech) [inline tests/Unit/Subscriber/Core/RegistratorSubscriberTest.php]: not stop

**Comment by dantleech** [comment]
Does this PR add the option? https://github.com/sulu-io/sulu-document-manager/pull/46

Did that PR not work?

**Comment by danrot** [comment]
No, it fixes a bug if this option is set.

**Comment by wachterjohannes** [comment]
TODO: add CHANGELOG.md file.

**Pull request merged**